movie/settings.py

```diff
@@ -0,0 +1,125 @@
+"""
+Django settings for movie project.
+
+Generated by 'django-admin startproject' using Django 5.1.3.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-cw_-i4uxi&t*+8sqlps7dyyq18r6f)ok^*_%*66ozjg%c&+orc'
```

Comment:
The `SECRET_KEY` should not be hardcoded in the settings file for production. Consider using environment variables to manage sensitive information securely.